ionic2/app/api/api.ts

```diff
@@ -0,0 +1,134 @@
+import { Injectable } from '@angular/core';
+import { Http, Response } from '@angular/http';
+import { Observable } from 'rxjs/Observable';
+
+import { Pokemon } from './models/pokemon';
+import { PokemonIcon } from './models/pokemon-icon';
+import { Sighting } from './models/sighting';
+
+export { Pokemon, PokemonIcon, Sighting}
+
+@Injectable()
+export class Api {
```

Comment:
This was done following the Ionic standards. They name their pages without the .component.ts.
It's up to preference, I just think that it makes sense since the angular router displays components on the page while the Ionic Navigation pushes and pops pages. I would've preferred if they would have followed A2 style aswell though. Can't give you a clear preference, sorry...